src/components/reviewMatches/NotificationWorklist.tsx

```diff
@@ -55,11 +58,11 @@ const columns: GridColDef[] = [
     align: 'center',
     headerAlign: 'center',
     valueFormatter: (params: GridValueFormatterParams<number>) =>
-      params.value ? moment(params.value).format('YYYY-MM-DD') : null
+      params.value ? moment.unix(params.value).format('DD/MM/YYYY') : null
```

Comment:
Would be better to transform any string to Date object at the fetch function level.
Then we could add a helper function to format any date, so we keen consistent format across the screens.